Rakefile

```diff
@@ -76,26 +76,6 @@ namespace :test do
     end or fail 'Failures'
   end
 
-  task :accept_license do
-    FileUtils.mkdir_p(File.join(Dir.home, '.chef', 'accepted_licenses'))
-    # If the user has not accepted the license, touch the acceptance
-    # file, but also touch a marker that it is only for testing.
-    unless File.exist?(File.join(Dir.home, '.chef', 'accepted_licenses', 'inspec'))
-      puts "\n\nTemporarily accepting Chef user license for the duration of testing...\n"
-      FileUtils.touch(File.join(Dir.home, '.chef', 'accepted_licenses', 'inspec'))
-      FileUtils.touch(File.join(Dir.home, '.chef', 'accepted_licenses', 'inspec.for_testing'))
-    end
-
-    # Regardless of what happens, when this process exits, check for cleanup.
-    at_exit do
-      if File.exist?(File.join(Dir.home, '.chef', 'accepted_licenses', 'inspec.for_testing'))
-        puts "\n\nRemoving temporary Chef user license acceptance file that was placed for test duration.\n"
-        FileUtils.rm_f(File.join(Dir.home, '.chef', 'accepted_licenses', 'inspec'))
-        FileUtils.rm_f(File.join(Dir.home, '.chef', 'accepted_licenses', 'inspec.for_testing'))
-      end
-    end
-  end
```

Comment:
Are you intending for devs to have just accepted the license and use our home folders? The ENV above is only for travis, no?

Reply:
Correct. Thats the pattern I have used so far. It does require accepting the license in all the CI systems (Travis, Appveyor, Buildkite, etc...)

I think this is somewhat of a philosophical question. We could certainly set `CHEF_LICENSE` in the `spec_helper.rb` (or somewhere else top level that will apply to all the tests). That would ensure the tests pass in all the CI systems but would also be accepting the license if someone ran the InSpec tests locally (which is approved in the EULA).

As I type all that out... maybe it is better to do it in the test setup/teardown rather than the CI system. Especially if we set `accept-no-persist` it would only accept it for that test run. Thoughts @zenspider @tas50 ?

Reply:
I think we're gonna have to do something more resilient because a number of our tests (and it should be more) set the HOME to a fake/mock directory.

I don't have an opinion about going the env route vs setup/teardown. The former is probably easier tho.

Reply:
Also, while I'm not fond of the code you're removing above, I'm a big proponent that anyone should be able to check out the code, install deps, run rake, and see green. This violates that idea... but I'm not sure what the right balance is.